features/step_definitions/payment_steps.rb

```diff
@@ -0,0 +1,14 @@
+Then("I should be on the {string} page") do |page|
+  carts_path
```

Comment:
This step does not do anything. No capybara command or assertion is present. ??

Reply:
Yes I can see it now, I'm changing it having a bit of a trouble how to find the right path for this cart